### PR TITLE
Clean up TGUI installation, install Prettier

### PR DESCRIPTION
**Pull request by dearmochi**
## What Does This PR Do

- Deletes unused `NtosHeader` and `NtosWindow`
- Removes `webpack-build-notifier`
	- I don't think this has any use at all considering our install builds quickly, it's really annoying
- Installs and runs Prettier rules (from tg) to enforce a semblance of consistency
	- Some ESLint rules had to be disabled to play nice

## Changelog
N/A

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -18,6 +18,12 @@
...
 	"[javascript]": {
-		"editor.defaultFormatter": "dbaeumer.vscode-eslint"
+		"editor.rulers": [80]
 	},
+	"[typescript]": {
+		"editor.rulers": [80]
+	},
+	"[scss]": {
+		"editor.rulers": [80]
+	}
```
Comment (moxian) [inline .vscode/settings.json:28]: may i pretty pretty please ask it to be 120, or *at least* 100?
80 is way way too claustrophobic. It's a relict of the 70s and is no longer relevant with modern screen sizes

**Review by moxian — COMMENTED** [review]
Does the vscode format thingy fix both prettier and eslint errors now, or just one or the other?.. Is the prettier enforced through the CI?

File: .vscode/settings.json (hunk 1 above)
Comment (AffectedArc07) [reply .vscode/settings.json:28]: Its just a ruler, this doesn't affect line length in CI checks

**Review by AffectedArc07 — APPROVED** [review]
What could go wrong

**Pull request merged**